app/helpers/application_helper/button/instance_migrate.rb

```diff
@@ -1,10 +1,6 @@
 class ApplicationHelper::Button::InstanceMigrate < ApplicationHelper::Button::Basic
-  def calculate_properties
-    super
-    self[:title] = @record.unsupported_reason(:live_migrate) if disabled?
-  end
-
   def disabled?
```

Comment:
Same here:

Maybe instead of this button, [GenericFeatureButtonWithDisable](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/application_helper/button/generic_feature_button_with_disable.rb) can be used